website/content/en/demos/standard-walkthrough/_index.md

```diff
@@ -45,7 +45,7 @@ Create an Azure Keyvault instance:
 Add a secret to your Keyvault:
 
 ```bash
-az keyvault secret set --vault-name ${KEYVAULT_NAME} --name secret1 --value "Hello!"
+az keyvault secret set --vault-name ${KEYVAULT_NAME} --name secret1 --value "Hello\!"
```

Comment:
is `\` intended in this secret value?

Reply:
It is! It's to escape the `!` that's in the double quotes.